src/runtime/runtime_stm32_timers.go

```diff
@@ -45,6 +47,10 @@ var (
 
 	// The timer used for sleeping
 	sleepTimer *timerInfo
+
+	// timer Prescaler and Auto-reload
+	timerPSC uint32
```

Comment:
It looks like timerPSC is only used inside one function, perhaps don't have as a global?